ReactAndroid/src/main/java/com/facebook/react/ReactNativeHost.java

```diff
@@ -58,6 +69,10 @@ public void clear() {
       mReactInstanceManager.destroy();
       mReactInstanceManager = null;
     }
+    if (mReactPerformanceLogger != null) {
+      mReactPerformanceLogger.release();
```

Comment:
What happens when app is reloaded in DEV mode ? Will the markers continue to aggregate ?